root_external_event_handling.tf

```diff
@@ -3,20 +3,35 @@ locals {
   event_handling_count                 = local.environment == "intg" ? 1 : 0
   sqs_name                             = "tdr-external-event-handling-sqs-${local.environment}"
   external_event_handler_function_name = "tdr-external-events-handler-${local.environment}"
+  dr2_ingest_topic                     = "${local.environment}-dr2-notifications"
   lambda_timeout                       = 60
 }
 
 module "external_event_handling_sqs_queue" {
-  count              = local.event_handling_count
-  source             = "./da-terraform-modules/sqs"
-  tags               = local.common_tags
-  queue_name         = local.sqs_name
-  sqs_policy         = templatefile("./templates/sqs/external_event_handling_policy.json.tpl", { region = local.region, environment = local.environment, account_id = data.aws_caller_identity.current.account_id, sqs_name = local.sqs_name })
+  count      = local.event_handling_count
+  source     = "./da-terraform-modules/sqs"
+  tags       = local.common_tags
+  queue_name = local.sqs_name
+  sqs_policy = templatefile("./templates/sqs/external_event_handling_policy.json.tpl", {
+    region           = local.region,
+    environment      = local.environment,
+    account_id       = data.aws_caller_identity.current.account_id,
+    sqs_name         = local.sqs_name,
+    dr2_account_id   = module.dr2_configuration.account_numbers[local.environment],
+    dr2_ingest_topic = local.dr2_ingest_topic
+  })
   encryption_type    = "kms"
   kms_key_id         = module.encryption_key.kms_key_arn
   visibility_timeout = 6 * local.lambda_timeout
 }
 
+resource "aws_sns_topic_subscription" "dr2_ingest_complete" {
```

Comment:
I would rename this: `dr2_ingest` It may or may not "complete" so best to keep it a bit less specific.